lib/Twig/Extension.php

```diff
@@ -76,4 +76,12 @@ public function getGlobals()
     {
         return array();
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getName()
+    {
+        return '';
```

Comment:
Imo yes, otherwise might lead to unexpected behaviour as you don't have to implement the method anymore, but third-party code might still deal with the returned value here (having several extensions returning empty strings might then mess things up).